airflow/utils/source_utils.py

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is probably only my opinion but `source_utils` is too ambiguous for me. Maybe `code_utils`?